tigerpath/templates/tigerpath/index.html

```diff
@@ -83,8 +83,6 @@
         </div>
         <div id="display-courses">
         </div>
-
-        <!-- Requirements -->
```

Comment:
Is this a remnant from something that was removed earlier?

Reply:
I think the requirements were probably there originally but got moved elsewhere